docker/pyramid/Dockerfile

```diff
@@ -34,6 +34,10 @@ RUN mkdir -p $REFERENCE_DATABASE_DIRECTORY
 
 RUN chown -R igvfd:igvfd $REFERENCE_DATABASE_DIRECTORY
 
+RUN mkdir -p /static/json
```

Comment:
I think maybe we want to use the package static folder instead.

Reply:
See comment below. And on ticket.